### DISCUSSION TITLE
Custom triggers

### DISCUSSION THREAD

**Discussion by felipecrs** [basnijholt/adaptive-lighting#409] [Ideas]
This integration is awesome, but I wonder if we should not extend it to support triggers other than just `sun`. For example:

- Keep some light on while a garage door is open.
- Keep some light on when a person is detected in camera.
- Change light RGB color to red when internet connectivity is lost (ping integration)

For all the examples above, leveraging the _Take back control_ option.

And things like that.

**Comment by th3w1zard1** [comment]
See the v2 roadmap for the main discussion, this has been asked many times.
